armi/reactor/tests/test_reactors.py

```diff
@@ -210,7 +208,7 @@ def loadTestReactor(
     if isPickeledReactor:
         # cache it for fast load for other future tests
         # protocol=2 allows for classes with __slots__ but not __getstate__ to be pickled
-        TEST_REACTOR = cPickle.dumps((o, o.r, assemblies.getAssemNum()), protocol=2)
+        TEST_REACTOR = cPickle.dumps((o, o.r, o.r.p.maxAssemNum), protocol=2)
```

Comment:
I think it might be quite a headache to change, but I'm guessing that `o.r.p.maxAssemNum` no longer really needs to be a part of this, because it will be encompassed within `o.r` already, right?

Reply:
Oh, for sure. 

But a quick check says that one change breaks over 500 unit tests in downstream repositories.

So... it's a fine idea, but I would want to make that it's own PR, as it would be mostly an exercise in communication and kitten herding.

Happily, that is just cleaning up our tests, and can't affect anyone's runtime.